aliasing/helpers.py

```diff
@@ -231,7 +231,7 @@ def set_cvar(character, name, value):
     value = str(value)
     if not name.isidentifier():
         raise InvalidArgument(
-            "Cvar names must be identifiers " "(only contain a-z, A-Z, 0-9, _, and not start with a number)."
```

Comment:
What even... happened here?

Reply:
The non-preview run of `black` determined that the string was short enough to fit on one line, but didn't merge the two strings into a single string